### ISSUE TITLE
Make table filters keyboard-compatible in IE

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3084]
In the table, the ‘Year’ column header button does not work in IE 11. When browsing in Google Chrome keyboard-only users are able to select the button to toggle between the table data being presented in ascending or descending order. Ensure that all functionality is operable using standard keyboard commands. Keyboard users would expect to navigate to, and away from interactive elements using the TAB and arrow keys. Users would then expect to activate elements using the Enter key or Spacebar

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by Norric1Admin** [comment]
Tabbing to the table filters and using 'Enter' to sort them works on Chrome but not IE.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Make table filters keyboard-compatible in IE](https://trello.com/c/2RhRrhKB/608-make-table-filters-keyboard-compatible-in-ie)

**Comment by SavvasStephanides** [comment]
## Update 1

As mentioned in the meeting today, this seems to work on all major browsers apart from IE (tested in IE11 on Browserstack). It even works on Edge. This seems to be due to a line in Javascript which IE doesn't recognise. Probably a newer API that isn't supported in older browsers. I'll need to investigate further.

**Comment by Norric1Admin** [comment]
## Description
In the table, the ‘Year’ column header button does not work in IE 11. When browsing in Google Chrome keyboard-only users are able to select the button to toggle between the table data being presented in ascending or descending order. Ensure that all functionality is operable using standard keyboard commands (keyboard users would expect to navigate to, and away from interactive elements using the TAB and arrow keys).

Testing page: ___

## Screenshots
**N/A**

## Successfully tested on...
- [ ] Windows: IE11
- [ ] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by Norric1Admin** [comment]
Changing this to `AAA` as it still requires a fix medium-term, but is no longer needed for `AA` accreditation as long as we provide the alternative below.
___
I've spoken to DAC and they are willing to give a little. The table feature isn't accessible, but the downloadable Excel file is. As long as we do the following, the 'AA' rating will be confirmed.

This line needs to be added to the catch-all text of every published statistical indicator (might be best done through Python?).

TEMPLATE: "Select the following link to download [ indicator : title [FILE SIZE]]((https://sustainabledevelopment-uk.github.io/sdg-data/data permalink .csv) in an accessible format"

EXAMPLE: "Select the following link to download [1.2.1: Proportion of population living below the national poverty line, by sex and age [15KB]](https://sustainabledevelopment-uk.github.io/sdg-data/data/1-2-1.csv) in an accessible format"

**Comment by Norric1Admin** [comment]
This is being tested on https://sdg.mango-solutions.com/download-test-deploy/1-1-1/

We still need to merge all indicators into the branch, but the test passed at the required level.

**Comment by AnnCorp** [comment]
Closing as this is AAA and we are working to AA standard.
